src/omnibar/omnibar.spec.ts

```diff
@@ -1284,14 +1286,79 @@ describe('Omnibar', () => {
 
       getTokenFake = () => Promise.resolve(testTokenWithNotificationEntitlement);
 
-      loadOmnibar();
+      loadOmnibarWithNotifications();
 
       fireMessageEvent({
         messageType: 'get-token',
         tokenRequestId: 123
       });
     });
 
+    describe('handle notifications per svcid', () => {
+
+      function testSvcId(
+        svcId: string,
+        tokenSpyCalled: boolean,
+        notificationsInitialized: boolean,
+        done: DoneFn
+      ): void {
+        startTrackingSpy.and.callFake((refreshUserCallback: () => void) => {
+          refreshUserCallback();
+
+          setTimeout(() => {
+            let allArgsExpectation = expect(getTokenSpy.calls.allArgs());
+            if (!tokenSpyCalled) {
+              allArgsExpectation = allArgsExpectation.not;
+            }
+            allArgsExpectation.toContain(
+              [{
+                disableRedirect: true,
+                envId: 'envid',
+                leId: 'leid',
+                permissionScope: 'Notifications'
+              }]
+            );
+
+            let toastContainerExpectation = expect(toastContainerInitSpy);
+            if (!notificationsInitialized) {
+              toastContainerExpectation = toastContainerExpectation.not;
+            }
+            toastContainerExpectation.toHaveBeenCalled();
+
+            done();
+          });
+        });
+
+        loadOmnibar({
+          envId: 'envid',
+          leId: 'leid',
+          svcId
+        });
+
+        fireMessageEvent({
+          messageType: 'get-token',
+          tokenRequestId: 123
+        });
+      }
+
+      it('renxt', (done) => {
```

Comment:
`it` blocks start with "should" and describe the actions under test. I would combine these 4 tests into 1 and call it "should respect the notification settings for a given service ID."